part-3/pk-sim-creating-individuals.md

```diff
@@ -1,80 +1,83 @@
-* Right mouse click on **Individuals** in the **Building Block Explorer** and select Add Individual...
-* Use the shortcut **Ctrl+Alt+I**
+-   Click on **Individual** <img src="../assets/icons/Individual.svg" data-size="line"/> in the **Create** Group of the **Modeling** Tab or
+-   Right mouse click on **Individuals** in the **Building Block Explorer** and select **Add Individual...** or
+-   Use the shortcut **Ctrl+Alt+I**
 
 A dialog will pop up in which the properties of the individual can be specified. First, new individuals are initialized by giving them a **Name** in the respective input field. These names are then used to identify them when their properties are saved in the project and/or as templates. In addition, names are used for identification of individuals in the simulation. Second, the properties of individuals can be set or changed:
 
-![The Create Individual dialog. Here, the properties of a non-standard
-European individual are shown.](../assets/images/part-3/PKSim-Individuals-NewIndividual.png)
+![The Create Individual dialog. Here, the properties of a standard European individual are shown.](../assets/images/part-3/PKSim-Individuals-NewIndividual.png)
 
-The **Create Individual** building block window is subdivided into three tabs: **Biometrics, Anatomy & Physiology**, and **Expression**.
+The **Create Individual** window is subdivided into three tabs: **Biometrics, Anatomy & Physiology**, and **Expression**.
 
 ### Biometrics‌
 
 In the first drop-down menu you can choose from the following species:
 
-* Human <img src="../assets/icons/Human.svg" alt="" data-size="line">
-* Monkey <img src="../assets/icons/Monkey.svg" alt="" data-size="line">
-* Beagle <img src="../assets/icons/Beagle.svg" alt="" data-size="line">
-* Dog <img src="../assets/icons/Dog.svg" alt="" data-size="line">
-* Minipig <img src="../assets/icons/Minipig.svg" alt="" data-size="line">
-* Rat <img src="../assets/icons/Rat.svg" alt="" data-size="line">
-* Mouse <img src="../assets/icons/Mouse.svg" alt="" data-size="line">
-* Rabbit <img src="../assets/icons/Rabbit.svg" alt="" data-size="line">
-* Cat <img src="../assets/icons/Cat.svg" alt="" data-size="line">
-* Cattle <img src="../assets/icons/Cattle.svg" alt="" data-size="line">
+-   Human <img src="../assets/icons/Human.svg" data-size="line"/>
+-   Monkey <img src="../assets/icons/Monkey.svg" data-size="line"/>
+-   Beagle <img src="../assets/icons/Beagle.svg" data-size="line"/>
+-   Dog <img src="../assets/icons/Dog.svg" data-size="line"/>
+-   Minipig <img src="../assets/icons/Minipig.svg" data-size="line"/>
+-   Rat <img src="../assets/icons/Rat.svg" data-size="line"/>
+-   Mouse <img src="../assets/icons/Mouse.svg" data-size="line"/>
+-   Rabbit <img src="../assets/icons/Rabbit.svg" data-size="line"/>
+-   Cat (experimental)<img src="../assets/icons/Cat.svg" data-size="line"/>
+-   Cattle (experimental)<img src="../assets/icons/Cattle.svg" data-size="line"/>
```

Comment:
I would not add "experimental" in the docu. It is already stated in the software.